docs/getting_started/trivia_template.md

```diff
@@ -88,11 +86,15 @@ npm start
 yarn start
 ```
 
-If everything goes well, you should see some reassuring messages in the console.
+If everything goes well, you should see:
+```bash 
+info: Bot launched. Visit: http://localhost:3000
+``` 
+Should you get an error, please visit our [Slack channel](https://slack.botpress.io) or [open an issue](https://github.com/botpress/botpress/issues/new).
```

Comment:
Let's point the community to `https://help.botpress.io` instead of Slack going forward as we are trying to transition Slack to the Forum :)

Reply:
@SSmale